radixdlt/src/main/java/com/radixdlt/consensus/validators/ValidatorSet.java

```diff
@@ -36,11 +36,11 @@
  * as long as all validators sign.
  */
 public final class ValidatorSet {
-	private final Map<ECPublicKey, Validator> validators;
+	private final ImmutableBiMap<ECPublicKey, Validator> validators;
```

Comment:
Not clear to me why this needs to be a bi-map at this point. Is this for future use?

Reply:
yes, it's used to get set of validators and ensuring no duplicates.

Reply:
But the way this map is constructed means that it is not possible for there to be any duplicates in any case?